openfisca_france/parameters/prestations/transport/aide_permis_demandeur_emploi.yaml

```
@@ -0,0 +1,4 @@
+reference: http://www.bo-pole-emploi.org/bulletinsofficiels/instruction-pe-n2011-205-du-9-de.html?type=dossiers/2011/bope-n2011-112-du-9-decembre-201
+montant:
```

Il s'agit d'un plafond pour couvrir les frais réellement engagés et non d'une aide forfaitaire :

> Cette aide financière est plafonnée à 1 200 euros et est destinée à prendre en charge tout ou partie des frais que le demandeur d’emploi expose dans le cadre de l’apprentissage des règles d’acquisition du permis de conduire automobile.

```suggestion
montant_max:
```